config/interpolate_funcs_test.go

```diff
@@ -174,76 +175,52 @@ func TestInterpolateFuncCoalesce(t *testing.T) {
 	})
 }
 
-func TestInterpolateFuncDeprecatedConcat(t *testing.T) {
```

Comment:
I don't believe so, though I will check again. The intention was to remove the ability of `concat()` to concat strings into another string, since it would require the return type of the Concat function to be `ast.TypeAny` which would break everything.